test/unit/__snapshots__/Editing.test.snap.js

```diff
@@ -0,0 +1,9 @@
+/* @web/test-runner snapshot v1 */
+export const snapshots = {};
+
+snapshots["Editing Element updates an element's attributes on UpdateActionEvent"] = 
+`<Substation name="A2">
```

Comment:
I need to be a bit more pain in the ass here, :) sorry. When this snapshot shall give us a full picture we need to extend the element under test `Substation` here to have at least one more attribute preferably from another namespace to make sure with the snapshot that we do not erase existing attributes from the element with the update action.

Reply:
We can include for example drawing syntax attributes in the Substation element, like this:
```
<SCL version="2007" revision="B" xmlns="http://www.iec.ch/61850/2003/SCL">
  <Substation name="A1" desc="test substation" sxy:x="1" sxy:y="1"></Substation>
</SCL>
```
Is that something you'd consider appropiate for this test? If not please suggest an specific additional attribute you'd prefer to see there.

Reply:
Yes, this would very be fitting. I would add the namespace description to the SCL tag. Same as `xmlns="http://www.iec.ch/61850/2003/SCL"` but for the `sxy`.